src/modules/user/beneficiaries/sagas/beneficiariesResendPinSaga.ts

```diff
@@ -0,0 +1,29 @@
+// tslint:disable-next-line
+import { call, put } from 'redux-saga/effects';
+import { API, RequestOptions } from '../../../../api';
+import { getCsrfToken } from '../../../../helpers';
+import { alertPush } from '../../../index';
+import {
+    BeneficiariesResendPin,
+    beneficiariesResendPinData,
+    beneficiariesResendPinError,
+} from '../actions';
+
+const config = (csrfToken?: string): RequestOptions => {
+    return {
+        apiVersion: 'peatio',
+        headers: { 'X-CSRF-Token': csrfToken },
+    };
+};
+
+export function* beneficiariesResendPinSaga(action: BeneficiariesResendPin) {
+    try {
+        const { id } = action.payload;
+        yield call(API.patch(config(getCsrfToken())), `/account/beneficiaries/${id}/resend_pin`, action.payload);
```

Comment:
why **id** exist in request route and request body

Reply:
because BE works like this:
https://github.com/openware/peatio/blob/master/app/api/v2/account/beneficiaries.rb#L118

Reply:
sad(